src/index.html

```diff
@@ -4,9 +4,376 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+
+      <nav>
+        <ul class="header__list">
+          <li class="header__items">
+            <a href="#apple" class="header__link is-active">Apple</a>
+          </li>
+          <li class="header__items">
+            <a href="#samsung" class="header__link">Samsung</a>
+          </li>
+          <li class="header__items">
+            <a href="#phones" class="header__link">Смарфоны и телефоны</a>
+          </li>
+          <li class="header__items">
+            <a
+              href="#laptops"
+              class="header__link"
+              data-qa="nav-hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="header__items">
+            <a href="#gadgest" class="header__link">Гаджеты</a>
+          </li>
+          <li class="header__items">
+            <a href="#tables" class="header__link">Планшеты</a>
+          </li>
+          <li class="header__items">
+            <a href="#photos" class="header__link">Фото</a>
+          </li>
+          <li class="header__items">
+            <a href="#video" class="header__link">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="page">
+      <div class="card page__card" buttom="card" data-qa="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__photo"
+        >
+
+        <h2 class="card__title">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </h2>
+
+        <span class="card__code">
+          Код товара: 195434
+        </span>
+
+        <div class="card__reviews">
+          <div class="card__reviews-stars stars--4 stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
```

Comment:
<div class="card__reviews-stars stars--4 stars">
you can use without : card__reviewws-stars 
only stars stars--4